modules/zero-trust-manager-uninstall-resources.adoc

```diff
@@ -51,25 +37,25 @@ $ oc delete SpireAgent cluster
 $ oc delete SpireServer cluster
 ----
 
-.. Delete the Persistent Volume Claim (PVC) by running the following command:
+.. Delete the `ZeroTrustWorkloadIdentityManager` cluster by running the following command:
 +
 [source,terminal]
 ----
-$ oc delete pvc -l=app.kubernetes.io/managed-by=zero-trust-workload-identity-manager
+$ oc delete ZeroTrustWorkloadIdentityManager cluster
 ----
 
-.. Delete the CSI Driver by running the following command:
+.. Delete the Persistent Volume Claim (PVC) by running the following command:
```

Comment:
persistent volume claim doesn't need to be capitalized. None of the object references do, unless you're referring to the actual object name - and in that case it should be in backticks

FYI: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting